deploy/docker/.env

```
@@ -20,5 +20,6 @@ TAG=latest
 
 TZ=Asia/Shanghai
 DATABASE=postgresql
+SPRING_JACKSON_TIME_ZONE=GMT+8
```

The default timezone in ds is UTC.
```suggestion
SPRING_JACKSON_TIME_ZONE=UTC
```

---

Can you update the doc together?